src/style.css

```diff
@@ -0,0 +1,115 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.search-form {
+  width: 100%;
+  position: relative;
+  background: #ffff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-weight: 300;
+}
+
+.search-container-big {
+  margin-top: 25px;
+  margin-bottom: 20px;
+  position: relative;
```

Comment:
There is a small problem with this div, it does not take 100% of the height. This prevents a user from clicking anywhere in the div area to focus on the search bar.
You should add `height: 100%` and remove `margin-top`.
After this you will need to move the `.img` using margins.